### PR TITLE
chore: fixes github template typo

### PR DESCRIPTION
**Pull request by ecbrodie**
## Description

Fixes issue template typo.

## Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Documentation only

## Checklist

**Please check all that apply.**

- [ ] Storybook updated with examples of new functionality
- [ ] Storybook uses variable and realistic data (ex: short and long text)
- [ ] Docs updated with correct props and examples
- [ ] Updated and reviewed changes to storyshots
- [ ] e2e tests added for component interations
- [ ] jest tests added for component API that may not be captured with storyshots (change handlers, renderers etc)
- [ ] Accessibility (includes relevant tags, keyboard functionality, colour contrast)

## Before Merging

- [ ] Tested storybook deployment preview
- [ ] Tested docs deployment preview

### CODE REVIEW

**Review by matthewlyle — APPROVED** [review]

**Comment by ecbrodie** [comment]
@marboro92 @matthewlyle apparently I have merge rights....shall I press the big green button? Or leave that to you. I'm fine either way :)

**Comment by matthewlyle** [comment]
> @marboro92 @matthewlyle apparently I have merge rights....shall I press the big green button? Or leave that to you. I'm fine either way :)

I'll merge it since i'm here now anyway but in the future yeah feel free to merge something we approved. Normally we merge immediately after reviewing, I just forgot here :)

**Pull request merged**
